.github/ISSUE_TEMPLATE/introduce-sgid-dataset.md

```diff
@@ -83,6 +83,7 @@ assign yourself or someone to check that the dataset is live in its area. once v
 - [ ] Stewardship doc and sgid-index (@assigned on ``)
 - [ ] `SGID.META.AGOLItems` (@assigned on ``)
 - [ ] `SGID.META.ChangeDetection` (@assigned on ``)
+- [ ] `agol-validator` items (tags, thumbnail, etc) (@assgined on ``)
```

Comment:
**suggestion**: can we link to the agol validator repository and hopefully on the readme or somewhere it has the list of features? That could replace the `tags, thumbnail, etc` part of this checkbox?

Reply:
Good idea, implemented.